### ISSUE TITLE
Remove unecessary variable when followed by a return

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#392]
I don't know if this is a valid rule for `UnusedVariable`, maybe `UnecessaryVariable`, but:

```php
function returnString(): string {
    $string = 'foo';

    return $string;
}

// vs.

function returnString(): string {
    return 'foo';
}
```

**Comment by Majkl578** [comment]
Duplicate of https://github.com/slevomat/coding-standard/issues/373 (where you actually added 🎉).
